src/data_processing/netflix_to_wikidata.py

```diff
@@ -30,7 +30,7 @@ def construct_query(title, year):
         SERVICE wikibase:mwapi {
             bd:serviceParam wikibase:api "EntitySearch" ;
                             wikibase:endpoint "www.wikidata.org" ;
-                            mwapi:search "%s" ;
+                            mwapi:search "%(search_term)s" ;
                             wikibase:limit 1 ;
```

Comment:
Remove the limit from the search query so that we get multiple results that we can sift through. Otherwise, you don't get `Lilo and Stitch` the 2002 movie.

Reply:
Good catch! Done